eng/targets/ResolveReferences.targets

```diff
@@ -207,11 +207,6 @@
       <_ExplicitPackageReference Remove="@(_ExplicitPackageReference)" />
     </ItemGroup>
 
-    <Warning
```

Comment:
Not needed? Temporary?

Reply:
I just don't think this is a useful warning (anymore?). We don't warn when making breaking code changes in `main`, I don't think we need to warn for breaking reference changes either.